appinventor/appengine/src/com/google/appinventor/client/youngandroid/YoungAndroidFormUpgrader.java

```diff
@@ -275,6 +275,9 @@ private static void upgradeComponentProperties(Map<String, JSONValue> componentP
       } else if (componentType.equals("HorizontalArrangement")) {
         srcCompVersion = upgradeHorizontalArrangementProperties(componentProperties, srcCompVersion);
 
+      } else if (componentType.equals("HorizontalScrollArrangement")) {
+        srcCompVersion = upgradeHorizontalScrollArrangementProperties(componentProperties, srcCompVersion);
```

Comment:
I didn't even know HorizontalScrollArrangements and VerticalScrollArrangements had been added to App Inventor until reading this code. You learn something new every day!